Adaptaion for android

Rakonda opened the pull request.

Rakonda committed:
Tabs items in android

On android its not possible to scroll left and right between items tabs, also the bouncing option is not available without setting has-bouncing to True.

Rakonda committed:
Remove scroll bar on Tabs (android)

There is a visible scroll bar when scrolling between tabs, normally ionic add an overflow-x: scroll when scroll direction ion-scoll is x, so to override it, i added overflow-x: hidden !important to .slidingTabs and it gives a nice look to the tabs.

leoruhland commented:
Hello @Rakonda !

Thank you very much for sending the PR.
I had been through this problem before.
I'm going to do real tests with iOS and Android and then I can accept the PR.
Thank you again!

Rakonda committed:
Support for $translate plugin

New attribute slide-tabs-translatable (true/false), this attribute works with plugin angular-translate, so instead of putting an ion-slide-tab-label="Title here" it supports the translate key -> ion-slide-tab-label="KEY.title". and the directive will render the $translate.instant('KEY.title');
Also the directive is watching for any changes in default display language so it will re-render the KEY.title with new output.

Commit:
Revert "Support for $translate plugin"

This reverts commit 994287e204f045822926d63871de7e0e779b4881.

Rakonda commented:
Hi
You're welcome @leoruhland, i found the index option very helpful, also i'm working on new option as attribute to support $translate plugin so the tabs title become translatable using KEY instead of text but it still under tests.
There is a commit for that you can check it out.

sandor commented:
Hey Guys,

really, really sorry for posting here for @leoruhland but i really don't know how to contact you. I am using your sliding tabs in a small project. Woking nicely, but I am facing problems if I am using nested slides – I have to use some vertical slides in a horizontal slider. If I am doing this, the tabs are controlled by the vertical slides and the vertical slides by the tabs. Would really need some urgen help here :-)

Thanks & Cheers,

Sandor

(rozsa@sharksinn.com)

Elvincth commented:
Sorry for bothering you I really need a event that can update the view of the tab.

I am facing the same problem as https://github.com/JKnorr91/ion-slide-box-tabs/issues/26 ,
it will be great if you can update directive thanks a lot.

“Actually I am adding slides dinamically using ng-repeat, and whenever user “pull to refresh” the page, new tabs are added or removed from tabs. I’m controlling this from my controller, but when I change the variable which controls ng-repeat content, the new tabs added not appears, and removed tabs keep showing on previous tabs. How can I update this whenever I want?”

Thanks in advance
